src/app/public/src/modules/sidebar/sidebar.component.spec.ts

```diff
@@ -147,20 +142,45 @@ describe('StacheSidebarComponent', () => {
       }
     ];
 
-    component.ngOnInit();
-    fixture.detectChanges();
     expect(component.heading).toEqual('Foo');
     expect(component.headingRoute).toEqual('/home/foo');
   });
 
   it('should add a heading for the root level route', () => {
-    fixture.detectChanges();
+    component.routes = undefined;
     expect(component.heading).toEqual('Home');
     expect(component.headingRoute).toEqual('/');
   });
 
   it('should automatically add a link to the top-level page', () => {
-    fixture.detectChanges();
+    component.routes = undefined;
```

Comment:
Should no longer be necessary to set these to undefined.  May have to detectChanges instead.